config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
```

Comment:
We still have https://github.com/discourse/discourse/blob/main/config/initializers/new_framework_defaults_7_0.rb. Should we work on removing that file first before merging the Rails 7.1 upgrade?

Reply:
I can try and see how it goes, yes.

Reply:
Done! :grin: